server/api/controllers/webhook/controller.ts

```diff
@@ -2,46 +2,25 @@ import axios from 'axios';
 import moment from 'moment';
 import { Request, Response } from 'express';
 import l from '../../../common/logger';
+import allData from './getData';
 
-type Record = { boardId: number; authToken: string; mondayAuthToken: string };
-const database: Array<Record> = [
-  {
-    boardId: 1964029256,
-    authToken: process.env.HUDDLECO_AUTH_TOKEN || '',
-    mondayAuthToken: process.env.MONDAY_AUTH_TOKEN || '',
-  },
-  {
-    boardId: 2706715613,
-    authToken: process.env.SALLY_A_CURTIS_AUTH_TOKEN || '',
-    mondayAuthToken: process.env.MONDAY_AUTH_TOKEN || '',
-  },
-  {
-    boardId: 2890900002,
-    authToken: process.env.THECOACHINGDIRECTORY_AUTH_TOKEN || '',
-    mondayAuthToken: process.env.THECOACHINGDIRECTORY_MONDAY_AUTH_TOKEN || '',
-  },
-  {
-    boardId: 2890960150,
-    authToken: process.env.KRISTI_AUTH_TOKEN || '',
-    mondayAuthToken: process.env.THECOACHINGDIRECTORY_MONDAY_AUTH_TOKEN || '',
-  },
-];
+type Record = { board_ID: string; username: string; api_key: string };
 
 const query = `
   mutation (
-    $boardId: Int!,
+    $board_ID: Int!,
     $itemName: String!,
     $columnValues: JSON!
   ) {
     create_item (
-      board_id: $boardId,
+      board_id: $board_ID,
       item_name: $itemName, 
       column_values: $columnValues
     ) { id }
   }`;
 
-const getRecord = (token: string | undefined): Record | undefined =>
-  database.find(({ authToken }) => authToken === token);
+const getRecord = async (token: string | undefined) =>
+  (await allData).find(({ username }) => username === token);
```

Comment:
`allData` should really be a function that you call when you need it. This is very presumptuious.